### DISCUSSION TITLE
Peacock aborted

### DISCUSSION THREAD

**Discussion by libinbabu1991** [idaholab/moose#28890] [Q&A Getting Started]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hi,
I am running moose from ubuntu app installed on my windows. I have added peacock the the path as instructed on the moose website.
`export PATH=~/projects/moose/python/peacock:$PATH`
However when I try to open peacock with or without an input file, I am getting "aborted" message. 
I would like to know how can I fix this error.
Thanks!

**Comment by GiudGiud** [comment]
Hello

I think making Peacock work on WSL takes a little bit of work.
You should start with this advice
https://github.com/idaholab/moose/discussions/28784#discussioncomment-10879737
and get Paraview and other examples to work. Once that works we can look at peacock which relies on a lot of the same machinery
